contracts/modules/legacy/LegacyPooledStaking.sol

```diff
@@ -375,13 +379,23 @@ contract LegacyPooledStaking is IPooledStaking, MasterAwareV2 {
   }
 
   function withdraw(uint /*ignoredParam*/) external override whenNotPaused onlyMember noPendingBurns {
-    uint amount = stakers[msg.sender].deposit;
-    stakers[msg.sender].deposit = 0;
-    token().transfer(msg.sender, amount);
-    emit Withdrawn(msg.sender, amount);
+    _withdrawForUser(msg.sender);
```

Comment:
I made this 1 function; it's the same code for both and thus i can exclude the scheduled migrated stakers from withdrawing.

Seems like it's most straightforward to just not allow them this way we avoid any race conditions and treating special cases.

Any other member can go ahead and withdraw their NXM.